helm-chart/README.rst

```diff
@@ -56,3 +56,43 @@ To run tests on the deployment, use
 .. code-block:: console
 
     $ helm test --cleanup renku
+
+
+Upgrading
+---------
+Most information related to upgrading from one chart version to another is covered
+in the `values changelog file<https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md>`_.
+For upgrades that require some steps other than modifying the values files to be
+executed, we add some instructions here.
+
+Upgrading to 0.8.0
+******************
+This upgrade comes with an upgrade of the keycloak chart from `4.10.2` to `9.8.1`! For
+details on this upgrade check the dedicated section in the
+`the keycloak chart docs<https://github.com/codecentric/helm-charts/tree/master/charts/keycloak#upgrading>`_
+and the `keycloak docs<https://www.keycloak.org/docs/latest/upgrading/>`_.
+
+- Before starting, make sure to check out `the values changelog for this upgrade<https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md#upgrading-to-renku-080-includes-breaking-changes>`_
+  and update your values file accordingly.
+
+- The upgrade of keycloak will perform an irreversible database migration. It is therefore recommended
+  to back up your postgres volume before performing this upgrade.
+
+- If they have not been explicitly defined (and thus autocreated through a helm hook), make sure to get the value
```

Comment:
I would rephrase this to really highlight that the two passwords have to be backed up before deleting them and just mention that is not necessary later.
e.g.:
"[WARNING] Backup the secrets: `renku-keycloak-postgres.keycloak-postgres-password` and `keycloak-password-secret.keycloak-password` with the command 
[...]
This operation is not needed if the secrets were previously explicitly defined [...]"

Reply:
👍

Reply:
Tried to make things clearer.